templates/base/kernel-ci-base.jinja2

```diff
@@ -1,3 +1,5 @@
+{% set extra_kernel_args = "console_msg_format=syslog" %}
```

Comment:
OK I think that should work for Depthcharge too, I'll make a patch in `generic-depthcharge-tftp-ramdisk-boot-template.jinja2`.

Reply:
Thanks, merged into this PR now.